src/dissect/dissect.c

```diff
@@ -369,6 +369,17 @@ static int strv_pair_to_json(char **l, JsonVariant **ret) {
         return json_variant_new_array_strv(ret, jl);
 }
 
+static void strv_pair_print(char **l, const char *prefix) {
+        char **p, **q;
+
```

Comment:
strlen will segfault if `prefix` is null, and in other helpers when there's a prefix it's usually optional, so having an assert here would be good